protocol/README.md

```diff
@@ -25,147 +25,149 @@ Every TBDex message contains the following fields:
+| `id` | string    | Y        | A unique identifier for this offering.|
+| `pair` | string    | Y        | The currency pair being offered, in the format of `basecurrency_countercurrency`.|
+| `unitPrice` | string    | Y        | Price of 1 unit of base currency denominated in counter currency.|
+| `fee`   | string       | Y        | Base fee associated with this offering |
+| `min`   | string       | Y        | Minimum amount of counter currency that the counterparty (Alice) must submit in order to qualify for this offering.|
+| `max`   | string       | Y        | Maximum amount of counter currency that the counterparty (Alice) can submit in order to qualify for this offering.|
+| `presentationRequest`   | PresentationRequest    | Y        |  PresentationRequest which describes the credential needed to choose this offer.|
+| `payinInstruments`   | list[PaymentInstrument]    | Y        |  A list of payment instruments the counterparty (Alice) can choose to send payment to the PFI from in order to qualify for this offering.|
+| `payoutInstruments`   | list[PaymentInstrument]    | Y        |  A list of payment instruments the counterparty (Alice) can choose to receive payment from the PFI in order to qualify for this offering.|
+
+### Note on base and counter currency in `pair`
+There's an explicit directionality baked into the `pair` naming convention, which is `BaseCurrency_CounterCurrency`. Base Currency is the currency that the PFI is selling. Counter Currency is the currency that the PFI is willing to accept to sell the base currency (in other words, PFI is buying the Counter Currency). In trading terms, the side is always "SELL". 
+
+### `PaymentInstrument`
 | field            | data type | required | description                                                                                          |
 | ---------------- | --------- | -------- | ---------------------------------------------------------------------------------------------------- |
-| `id` | string    | Y        | A unique identifier for this request.                                                                 |
-| `have` | string    | Y        | What you have.                                                                 |
-| `want` | string    | Y        | What you want.                                                                           |
-| `side`   | enum       | Y        | One of ["have", "want"] |
-| `size`   | int       | Y        | When side is "want", amount of want to spend on order. When "have", amount of have to spend on order.  |
-| `paymentTypes`   | JSON Object    | N        |  A normalized object describing the type of payments you need quotes for. When absent, the PFI is free to include any payment types.                 |
+| `kind` | enum    | Y        | Type of payment instrument (i.e. `DEBIT_CARD`, `BITCOIN_ADDRESS`)|
+| `fee` | string    | N        | Optional fee associated with using this kind of payment instrument.|
```

Comment:
Should fee be a separate object? That way it can evolve easily to different fee structures (or amount vs %).

Reply:
yeah, i was thinking fee would get more complex over time. i'll change it to be an object that just contains `flatFee` key for now.